mqt/qcec/verify.py

```diff
@@ -28,6 +34,12 @@ def verify(
     :return: The results of the equivalence checking process.
     """
 
+    def is_parameterized(qc: QuantumCircuit) -> bool:
+        return not isinstance(qc, str) and qc.parameters
+
+    if is_parameterized(circ1) or is_parameterized(circ2):
+        return check_parameterized(circ1, circ2, configuration, **kwargs)
+
```

Comment:
Just a reminder: the `verify_compilation` function should also be enriched with these new options.
(At which point it would probably be great again to have the `is_parameterized` function available to both)

Reply:
Extracted it into a function in the `parameterized` module so both verify and verify_compilation can use it